tests/core/test_signal.py

```diff
@@ -78,8 +81,6 @@ async def test_signal_connect_fails_if_different_backend_but_same_by_value():
         await sim_signal.connect(mock=False, backend=MockSignalBackend(str))
     assert str(exc.value) == "Backend at connection different from initialised one."
 
-    await sim_signal.connect(mock=False, backend=initial_backend)
```

Comment:
Why remove this line? It's demonstrating you can connect with backeend so long as it's the same as initial_backend.